java/fury-core/src/test/java/org/apache/fury/util/StringUtilsTest.java

```diff
@@ -23,9 +23,11 @@
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
+import org.apache.fury.FuryTestBase;
+import org.apache.fury.memory.Platform;
 import org.testng.annotations.Test;
 
-public class StringUtilsTest {
+public class StringUtilsTest extends FuryTestBase {
```

Comment:
```suggestion
public class StringUtilsTest {
```

Reply:
`StringUtilsTest.testVectorizedLatinCheckAlgorithm(boolean)` asks for a `@Dataprovider` named "endian"

Reply:
I see, thanks